assignment.rb

```diff
@@ -0,0 +1,51 @@
+# 1.1 User Input Assignment
+
+puts "Hello. Please tell me your name."
+
+name_LCV = 1
+while name_LCV == 1
+  print ">> "
+  name = gets.chomp
+  print ""
```

Comment:
What is this extra `print` doing? I'm not sure it is doing anything noticeable, so if it's not supposed to be there, you could probably remove it.